### ISSUE TITLE
Styles -> Widgets -> Calender -> Colors -> Links is showing a white, blank sidebar

### ISSUE THREAD

**Issue by andreas2801** [WordPress/gutenberg#48734]
### Description

Color selection for element links for widget calender not possible (blank screen). 
See screenshot attached. 

### Step-by-step reproduction instructions

See screenshot attached

### Screenshots, screen recording, code snippet

![image](https://user-images.githubusercontent.com/5755586/222754722-b57fbdc5-7aa3-4f5a-b70d-69f3dbcc6ee1.png)
![image](https://user-images.githubusercontent.com/5755586/222755355-44819b76-4480-4d0b-95f5-6bedd2079d5c.png)

### Environment info

WP 6.1.1
Gutenberg-Plugin 15.2.4 

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by kishanjasani** [comment]
@andreas2801 I am not able to replicate the issue. I have tried to replicate it in multiple ways but cannot reproduce it. Could you please provide a screen recording or detailed steps to reproduce it?

I have tried to reproduce it using the Twenty Twenty-Three theme and Gutenberg plugin 15.3.0-rc.1
Check the screenshot: https://prnt.sc/5uzCeET6IzZh

Thanks

**Comment by andreas2801** [comment]
Hello @kishanjasani 

I attached a video of the blank screen. I´m new to the WP system and don´t know how I could better support you to localize the root of that behavier. Please, tell me if needed, how I could provide you more information to better enclose the root of that problem. 

https://user-images.githubusercontent.com/5755586/222901617-eef5d7c1-3581-4d9b-b417-c57c26b2543a.mp4

**Comment by ghost** [comment]
@andreas2801 This is working on my end with both the TT2 theme and the TT3 theme. Is the theme you're using extending the widget block in any way?

**Comment by andreas2801** [comment]
@tomdevisser I have currently, beside the Gutenberg-plugin, two plugins installed. Not sure if they do extend the widget block in any way. 
![image](https://user-images.githubusercontent.com/5755586/223192141-b5473e2d-613e-441f-b572-a9f0bf9463cc.png)

I did just for test activate them and the color for the calender links did suddenly appear. Then I did another test and deactivated the two plugins and the color sidebar is still working. Not sure what is happening there but now its working and I think this issue can be closed.

Thank you. 

ps: Here is the theme-folder
[daisyonwp-starter.zip](https://github.com/WordPress/gutenberg/files/10901121/daisyonwp-starter.zip)
